exporter/otlphttpexporter/otlp.go

```diff
@@ -142,7 +142,7 @@ func (e *exporterImp) export(ctx context.Context, url string, request []byte) er
 
 	defer func() {
 		// Discard any remaining response body when we are done reading.
-		io.CopyN(ioutil.Discard, resp.Body, maxHTTPResponseReadBytes)
+		io.CopyN(ioutil.Discard, resp.Body, maxHTTPResponseReadBytes) // nolint:errcheck
 		resp.Body.Close()
```

Comment:
require.NoError on close?

Reply:
golangci-lint excludes some errchecks by default since they are not usually checked, I will add error checking for those too

Reply:
@bogdandrutu I am not convinced that we should check for errors when closing a request body: they are not even checked on the https://golang.org/pkg/net/http example, and grepping on important Go codebases like the Go compiler or Kubernetes they don't seem to be ever checked. 

Happy to do it if you think we should check them but I am hesitant about it

Reply:
Happy to leave unchecked.